src/test/java/pixelperfect/SpaceshipTest.java

```diff
@@ -4,23 +4,32 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import org.junit.Before;
 import org.junit.Test;
 
-import pixelperfect.Spaceship;
-
 /**
  * Test Suite for the Spaceship class.
  * 
  * @author Jesse Tilro
  *
  */
 public class SpaceshipTest {
+  
+  private Spaceship ship;
+  
+  /**
+   * Initialize objects.
+   */
+  @Before
+  public void before() {
+    ship = new Spaceship();
+  }
+  
   /**
    * Test the udate health method.
    */
   @Test
-  public void updateTest() {
-    Spaceship ship = new Spaceship();
+  public void testUpdate() {
```

Comment:
But it will test everything that happens in the update method so it won't be only about health. I guess it is just better to have a test for each part in the update method. (could also have been an integration test)